libs/components/ag-grid/src/lib/modules/ag-grid/types/lookup-properties.ts

```diff
@@ -9,27 +9,106 @@ import {
 } from '@skyux/lookup';
 
 export interface SkyAgGridLookupProperties {
+  /**
+   * Fires when users select the button to add options to the list.
+   */
   addClick?: (args: SkyLookupAddClickEventArgs) => void;
+  /**
+   * The `aria-label` text for the lookup cell. If neither `ariaLabel` or `ariaLabelledBy` are specified, the `aria-label` defaults to the column's `headerName`, `headerTooltip`, `field`, or `colId`.
+   */
   ariaLabel?: string;
+  /**
+   * The HTML element ID of the element that labels the lookup cell. If neither `ariaLabel` or `ariaLabelledBy` are specified, the `aria-label` defaults to the column's `headerName`, `headerTooltip`, `field`, or `colId`.
+   */
   ariaLabelledBy?: string;
-  /** @deprecated */
+  /**
+   * The value for the autocomplete attribute on the form input.
+   * @default "off"
+   * @deprecated
+   */
   autocompleteAttribute?: string;
+  /**
+   * The data source for the lookup cell to search when users enter text. You can specify static data such as an array of objects, or you can pull data from a database.
```

Comment:
```suggestion
   * The data source for the lookup cell to search when users enter text. You can specify static data, such as an array of objects, or you can pull data from a database.
```
Also, the corresponding property description for `autocomplete-properties.ts` includes a sentence that says "For a dynamic data source such as an array that changes due to server calls, use search instead." Should we have something like that here too?

Reply:
Since the property is already deprecated in favor of `searchAsync` I think we don't need that addition. Autocomplete cell doesn't support `searchAsync` yet so the `search` suggestion is more relevant